caom2-ui-server/build.gradle

```diff
@@ -12,19 +12,20 @@ repositories {
 
 dependencies {
     compile 'log4j:log4j:[1.2.17,)'
+    compile 'com.opencsv:opencsv:[3.10,)'
```

Comment:
I don't think this is used, so I removed it.  Did you confirm whether it's necessary?

Reply:
no, just part of merging issues.